btrfsmaintenance-refresh-cron.sh

```diff
@@ -58,7 +58,12 @@ refresh_timer() {
 	echo "Refresh timer $SERVICE for $PERIOD"
 
 	case "$PERIOD" in
-		daily|weekly|monthly)
+		uninstall)
```

Comment:
I would add "none" here, as it's already used but is not a valid value for the OnCalendar directive.
This would avoid errors in logs such as "Failed to parse calendar specification, ignoring: none".

Reply:
Makes sense, please send a patch.